client/tests/sdk/test_evaluation.py

```diff
@@ -351,6 +352,19 @@ def predict(self, data: t.Any, external: t.Any) -> t.Any:
                 assert data.label == 1
                 assert "index" in external
 
+        class DummyWithDecoratorOnlyData(PipelineHandler):
+            @http_retry(attempts=3)
+            def predict(self, data: t.Any) -> t.Any:
+                assert data.label == 1
+
+        class DummyWithDecoratorKeyword(PipelineHandler):
```

Comment:
Would it hava a problem when user coding as this:
`class DummyWithDecoratorKeyword(PipelineHandler):
            def init(self) -> None:
                 self.name = 'test'
            @http_retry(attempts=3)
            @pass_context
            def predict(self, data: t.Any, **kw: t.Any) -> t.Any:
                assert self.name == 'test'